rsts/concepts/execution_timeline.rst

```diff
@@ -5,3 +5,62 @@ Timeline of a workflow execution
 ########################################
 
 .. image:: https://raw.githubusercontent.com/lyft/flyte/assets/img/flyte_wf_timeline.svg?sanitize=true
+
+
+The illustration above refers to a simple workflow, with 2 nodes N1 & N2. This can be represented as follows,
+
+.. mermaid::
+
+    graph LR;
+         Start --> N1;
+         N1 --> N2;
+         N2 --> End;
```

Comment:
The graph is create for a visual can we add to, or make another graph showing the timeline of latencies? This may be difficult to capture, but it could be very useful.

graph LR;
    FlyteAdmin execution Request --Acceptance Latency--> FlytePropeller Evaluation
    FlytePropeller Evaluation --EvaluationLatency-->FlytePropeller Evaluation
    FlytePropeller Evaluation --Completion Latency-->Success
    ...

Reply:
that is actually captured in the Timeline view - did you see that?